core/services/pipeline/task.eth_call.go

```diff
@@ -79,6 +80,7 @@ func (t *ETHCallTask) Run(ctx context.Context, lggr logger.Logger, vars Vars, in
 		errors.Wrap(ResolveParam(&gasTipCap, From(VarExpr(t.GasTipCap, vars), t.GasTipCap)), "gasTipCap"),
 		errors.Wrap(ResolveParam(&gasFeeCap, From(VarExpr(t.GasFeeCap, vars), t.GasFeeCap)), "gasFeeCap"),
 		errors.Wrap(ResolveParam(&chainID, From(VarExpr(t.EVMChainID, vars), NonemptyString(t.EVMChainID), "")), "evmChainID"),
+		errors.Wrap(ResolveParam(&gasUnlimited, From(NonemptyString(t.GasUnlimited), false)), "gasUnlimited"),
```

Comment:
If you don't use `VarExpr` then the user won't be able to use expressions like `$(lessThanTask)` in job spec.

Reply:
done